store-implementation/parquet-store/src/test/java/uk/gov/gchq/gaffer/parquetstore/ParquetStoreTest.java

```diff
@@ -39,4 +88,36 @@ public void testTraits() throws StoreException {
         expectedTraits.add(StoreTrait.VISIBILITY);
         assertEquals(expectedTraits, store.getTraits());
     }
+
+    @Test
+    public void shouldNotThrowExceptionWhenAddingASingleEdgeWithGroupNotInSchema() throws Exception {
+        getGraph().execute(new AddElements.Builder()
+                        .input(testEdge)
+                        .build(),
+                new User());
+    }
+
+    @Test
+    public void shouldNotThrowExceptionWhenAddingASingleEntityWithGroupNotInSchema() throws Exception {
+        getGraph().execute(new AddElements.Builder()
+                        .input(testEntity)
+                        .build(),
+                new User());
+    }
+
+    @Test
+    public void shouldNotThrowExceptionWhenAddingAMultipleElementsWithGroupsNotInSchema() throws Exception {
+        getGraph().execute(new AddElements.Builder()
+                        .input(testEntity, testEdge)
+                        .build(),
+                new User());
+    }
+
```

Comment:
Can we have one more test to cover the case when one element is not in the schema, and one is in the schema - in this case the one that is in the schema should still be added.